commands/init.go

```diff
@@ -45,13 +45,14 @@ func transformInitArgs(args *Args) error {
 		err    error
 	)
 
-	if args.IsParamsEmpty() {
+	words := args.Words()
```

Comment:
I missed this case. I've fixed it by parsing all flags and arguments for `git-init` to get the optional directory argument.